client/src/components/Workflow/Invocations.vue

```diff
@@ -120,6 +119,7 @@ export default {
     },
     methods: {
         ...mapCacheActions(["fetchWorkflowForInstanceId", "fetchHistoryForId"]),
+        ...mapActions("betHistory", ["setCurrentHistoryId"]),
```

Comment:
```suggestion
        ...mapActions("betaHistory", ["setCurrentHistoryId"]),
```

Reply:
(you probably saw this, but just commenting so it doesn't bite you later if you didn't)

Reply:
Yup, thanks!